setup.cfg

```diff
@@ -33,7 +33,7 @@ ignore_missing_imports = True
 
 [flake8]
 max-line-length = 200
-exclude = venv/*,var/*,alembic/*,frontend/*,codalab/lib/beam/ratarmount.py
+exclude = venv/*,var/*,alembic/*,frontend/*,codalab/lib/beam/ratarmount.py,codalab/lib/beam/streamingzipfile.py
```

Comment:
I'm excluding streamingzipfile.py from black / flake8 because much of the code in it comes from and is written in the style of https://github.com/python/cpython/blob/3.9/Lib/zipfile.py. Eventually, we may want to contribute a PR back to cpython that adds this feature in to zipfile.